### PR TITLE
PJSIP 2.8, openh284 1.8, opus 1.3.1 & iOS SDK 12.2

### PR DESCRIPTION
**Pull request by welljsjs**
Upgraded sources (#61)

I think one could consider using `VideoToolbox` framework instead of `H.264`, which will improve hardware acceleration (also means less third-party dependencies).

Additionally, I also think that it's completely possible to support iOS and macOS with a single pod. Just look at [OpenSSL-Universal](https://github.com/krzyzanowskim/OpenSSL) and how they deal with that. I'll try to build something similar for pjsip within the next days or weeks.

@chebur (sorry to wake you up), but is this repo still maintained/do PRs get reviewed and accepted? 
Otherwise, I'd consider publishing my own version of this.

### CODE REVIEW

File: openh264.sh [hunk 1]
```diff
@@ -37,7 +37,7 @@ function build() {
     fi
     mkdir -p ${PREFIX}
 
-    echo "Builing for ${ARCH}..."
+    echo "Building for ${ARCH}..."
```
Comment (welljsjs) [inline openh264.sh]: And, uh, just fixed this typo.

File: pjsip.podspec [hunk 2]
```diff
@@ -59,7 +59,7 @@ LICENSE
   }
 
   s.dependency          'OpenSSL-Universal'
-  s.frameworks          = 'CFNetwork', 'AudioToolbox', 'AVFoundation', 'CoreMedia'
```
Comment (welljsjs) [inline pjsip.podspec]: I dunno whether `'CFNetwork', 'AudioToolbox', 'AVFoundation', 'CoreMedia'` are still required. Haven't tested without them yet.

**Comment by chebur** [comment]
Hi @welljsjs. Currently I have no free time to maintain the pod by myself. I really appreciate your work and will review the PR and hope to merge it soon

**Comment by welljsjs** [comment]
Having a little issue at the moment because pjsip ≥ 2.6 expects to be built with OpenSSL-1.1.x. [That's at least what their docs say.](https://trac.pjsip.org/repos/wiki/Getting-Started/iPhone#OpenSSLSupport)

I don't know how  pjsip ≥ 2.6 behaves with OpenSSL-1.0.x, although I'm getting no errors when building. 

Unfortunately, [the OpenSSL-for-iPhone repo](https://github.com/x2on/OpenSSL-for-iPhone) does not seem to support OpenSSL-1.1.x versions according to their build script:
 > Note: This script does not yet work with OpenSSL 1.1.0

Also, I'm now switching to [OpenSSL-Universal ](https://github.com/krzyzanowskim/OpenSSL), which is a requirement for building for macOS in order to create a multi-platform pod.

**Pull request merged**